.devcontainer/docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+chown -R node:node "${THELOUNGE_HOME}"
+
+sleep infinity
```

Comment:
ehm, why?

Reply:
this dir needs to be owned by the same user running the container so that it doesnt become owned by root on the host when mounting into the container.

Reply:
doesn't matter if it's owned by root on the host, if you use namespace mapping properly (as in set up uid maps) root in the container will not be uid 0 on the host.
And all example containers of vscode get away without any such chowning or in fact any entryfile at all

Reply:
the container is being run by user `node` with id of `1000`, but because docker is creating the config file on start of the container if it doesnt exists(a fresh dev setup) its created as root. this is what is done in the official docker: https://github.com/thelounge/thelounge-docker/blob/cfd776abbf841dc561608e4765b3774d24562232/docker-entrypoint.sh#L4